Source/WebCore/platform/graphics/displaylists/DisplayListItems.h

```diff
@@ -951,34 +951,23 @@ std::optional<DrawLinesForText> DrawLinesForText::decode(Decoder& decoder)
 class DrawDotsForDocumentMarker {
 public:
     static constexpr ItemType itemType = ItemType::DrawDotsForDocumentMarker;
-    static constexpr bool isInlineItem = true;
+    static constexpr bool isInlineItem = false;
```

Comment:
why was this changed?

Reply:
See ChangeLog:

> `DrawDotsForDocumentMarker` can no longer be an inline item, since `Color` can
have out-of-line components.